static/templates/subscription_settings.handlebars

```diff
@@ -65,9 +65,12 @@
                     </li>
                     <li>
                         <div id="sub_push_notifications_setting"
-                          class="sub_setting_checkbox sub_notification_setting {{#unless in_home_view}}muted-sub{{/unless}}">
-                            <input id="push-notifystream-{{stream_id}}" class="sub_setting_control" type="checkbox" tabindex="-1" {{#if push_notifications}}checked{{/if}} {{#unless in_home_view}}disabled="disabled"{{/unless}}/>
+                          class="sub_setting_checkbox sub_notification_setting {{#unless in_home_view}}muted-sub{{/unless}} {{#unless page_params.realm_push_notifications_enabled}}control-label-disabled{{/unless}}">
```

Comment:
@pragatiagrawal31 I want you to take a closer look at this:
```js
    $("#subscriptions_table").on("click", "#sub_push_notifications_setting",
                                 stream_push_notifications_clicked);
```
this will explain that unusual toggling behavior about which we had the discussion, and reason why that handler removal doesn't affect the toggling on this `div`.

Reply:
Hint: Turn off the server :smile: